Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp: add include for WPE_*_VERSION
Source/WebKit/UIProcess/gtk/AcceleratedBackingStoreWayland.cpp: add include for wpe_view_backend_initialize()

--- webkitgtk-2.30.5/Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp.orig	2020-10-23 10:24:22.000000000 +0200
+++ webkitgtk-2.30.5/Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp	2021-03-13 16:00:53.328034094 +0100
@@ -43,6 +43,7 @@
 
 #if PLATFORM(WAYLAND) && USE(WPE_RENDERER)
 #include <wpe/fdo.h>
+#include <wpe/wpe.h>
 #endif
 #endif
 
--- webkitgtk-2.30.5/Source/WebKit/UIProcess/gtk/AcceleratedBackingStoreWayland.cpp.orig	2020-09-11 10:16:51.000000000 +0200
+++ webkitgtk-2.30.5/Source/WebKit/UIProcess/gtk/AcceleratedBackingStoreWayland.cpp	2021-03-13 16:14:40.706885131 +0100
@@ -50,6 +50,7 @@
 #endif
 
 #if USE(WPE_RENDERER)
+#include <wpe/wpe.h>
 #include <wpe/fdo-egl.h>
 #if WPE_FDO_CHECK_VERSION(1, 7, 0)
 #include <wayland-server.h>
